### ISSUE TITLE
Kin'iro no Corda 4 [PCSH00275]

### ISSUE THREAD

**Issue by Croden1999** [Vita3K/compatibility#2298]
# App summary
- App name: 金色琴弦４
- App serial: PCSH00275
- App version: 1.00

# Vita3K summary
- Version: v0.1.8
- Build number: 3320
- Commit hash: https://github.com/vita3k/vita3k/commit/962b1f8a
- CPU backend: Dynarmic
- GPU backend: Vulkan

# Test environment summary
- Tested by: Croden
- OS: Windows 10
- CPU: Intel Core i7-7700HQ
- GPU: NVIDIA GeForce GTX 1050
- RAM: 16 GB

# Issues
Could not booting, and also crash

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/12330400/vita3k.log)

# Recommended labels
- Nothing

**Comment by othmanead** [comment]
disable ngs and try again

**Comment by Zangetsu38** [comment]
Game works fine, but request disable NGS

**Comment by Croden1999** [comment]
> Game works fine, but request disable NGS

yes, u r right sir, disable can play it, small problems back to the title menu get stuck

![image](https://github.com/Vita3K/compatibility/assets/61804715/ac2a1b90-2e16-4530-823c-91151b76cf96)

**Comment by potatoteatime** [comment]
Tested with 0.1.9 Build 3338 and with NGS disabled, game can start without audio, but with the latest ver (0.2.0), game starts with black screen and then crashes even with NGS disabled.

**Comment by github-user1112** [comment]
# Vita3K summary
Version: v0.2.1
Build number: 3967
Commit hash: https://github.com/Vita3K/commit/9916d86
GPU backend: Vulkan

# Test environment summary
Tested by: gtihub-user
OS: Linux Mint 22.3
CPU: AMD Ryzen 5 5600
GPU: Radeon Rx 6600
RAM: 16 GB

# Issues
Still nothing and crashes with NGS on
